internal/util/context.go

```diff
@@ -0,0 +1,28 @@
+package util
```

Comment:
cc: @grayside can you lend us a review for this small file?
we've hacked a way to propagate the logger with the ctx.

Reply:
👍 LGTM

Maybe this should be package internal/logger?